bitsail-connectors/bitsail-connectors-legacy/bitsail-connector-kafka/src/test/java/com/bytedance/bitsail/connector/legacy/kafka/source/KafkaSourceITCase.java

```diff
@@ -59,34 +68,87 @@ private static String constructARecord(int index) {
   public void before() {
     kafkaCluster.startService();
     kafkaCluster.createTopic(topicName);
-    startSendDataToKafka();
   }
 
-  private void startSendDataToKafka() {
+  private void startSendJsonDataToKafka() {
     KafkaProducer<String, String> producer = kafkaCluster.getProducer(topicName);
     ScheduledThreadPoolExecutor produceService = new ScheduledThreadPoolExecutor(1);
     AtomicInteger sendCount = new AtomicInteger(0);
     produceService.scheduleAtFixedRate(() -> {
       try {
-        for (int i = 0; i < 5000; ++i) {
-          String record = constructARecord(sendCount.getAndIncrement());
-          producer.send(new ProducerRecord(topicName, record));
-        }
+	for (int i = 0; i < 5000; ++i) {
+	  String record = constructARecord(sendCount.getAndIncrement());
+	  producer.send(new ProducerRecord(topicName, record));
+	}
       } catch (Exception e) {
-        LOG.error("failed to send a record");
+	LOG.error("failed to send a record");
       } finally {
-        LOG.info(">>> kafka produce count: {}", sendCount.get());
+	LOG.info(">>> kafka produce count: {}", sendCount.get());
+      }
+    }, 0, 1, TimeUnit.SECONDS);
+  }
+
+  @SneakyThrows
```

Comment:
not sure why we need this `SneakyThrows` here, would you elaborate？

Reply:
Removed. Thanks for pointing out!